addons/payment_stripe/static/src/js/stripe.js

```diff
@@ -39,7 +40,12 @@ odoo.define('payment_stripe.stripe', function(require) {
         if(!$(this).find('i').length)
             $(this).append('<i class="fa fa-spinner fa-spin"/>');
             $(this).attr('disabled','disabled');
-        handler.open();
+        handler.open({
+            name: $("input[name='merchant']").val(),
+            description: $("input[name='invoice_num']").val(),
```

Comment:
I just tested the ecommerce and SO number is there.

About `invoice_num` I think it was improperly named before my PR. For our usage would make more sense show SO to user. If OK with you i'll change the name to `description` or `reference`, just in case in the future we will accept invoice payments?